components/SocialButton/SocialButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Button from '../Button';
+import FacebookIcon from './sub-components/FacebookIcon';
+import GoogleIcon from './sub-components/GoogleIcon';
+import { theme as defaultTheme } from '../shared';
+
+const GoogleButton = styled(Button)`
```

Comment:
Suggestion: to reuse and center code, how about this:

```
const BUTTON_SIZE = '40px';
const BUTTON_PADDING = defaultTheme.spacing.xsmall;

const BaseButton = styled(Button)`
  width: ${BUTTON_SIZE};
  height: ${BUTTON_SIZE};
  border-radius: 9px;
  cursor: pointer;
  overflow: hidden;
`;

const GoogleButton = styled(BaseButton)`
  outline: none;
  padding: ${BUTTON_PADDING - 1}px;
`;

const FacebookButton = styled(BaseButton)`
  border: none;
  background-color: #3c5193;
  padding: ${BUTTON_PADDING}px;
`;
```

Reply:
Yes, I liked this suggestion. Gonna adopt it, thanks!